packages/metadata-common/src/AddMetadataButton.tsx

```diff
@@ -82,8 +82,8 @@ export const AddMetadataButton = (
           }
           title={`Create new ${
             singleSchema
-              ? props.schemas?.[0].display_name
-              : props.schemas?.[0].schemaspace
+              ? props.schemas?.[0].title
+              : props.schemas?.[0].schemaspace.slice(0, -1)
```

Comment:
Removed the trailing 's' from string since all schemaspaces are plural

Reply:
This won't work for plural forms that introduce `es` or `ies` (or in old Rome `ae` or `i` :smile: ).

Instead, I think it might be best to add a `title` property definition to `meta-schema.json` and mark it as required.  That way the validation performed against BYO schemas (via the meta-schema) will ensure that a `title` property exists and we shouldn't need this tertiary statement.  It should be fairly open wrt constraints - perhaps something like this:
```JSON
    "title": {
      "description": "The displayable title of the schema.",
      "type": "string"
    },
```

Reply:
As per our discussion, I updated all metadata widgets to use `schemaType` property, which will be appended to the `AddMetadataButton` hover text